src/renderer.ts

```diff
@@ -54,10 +54,10 @@ export class Renderer {
             // find best scoring section editor combo (or undefined if too low score)
             const topSection = this.topSectionFinder.findTopSectionForEditor(textEditor, sections);
 
-            if (!topSection) { return ; }
-
-            this.filterCoverage(topSection, coverageLines);
-            this.setDecorationsForEditor(textEditor, coverageLines);
+            if (topSection) {
```

Comment:
@ryanluker this can cause the wrong score, etc. to carry over because it doesn't flush properly.  noticed with the Python tests with the score filtering thing.